code/modules/mob/living/living_defense.dm

```diff
@@ -186,6 +186,8 @@
 		return 1
 	if(fire_stacks > 0)
 		adjust_fire_stacks(-0.1) //the fire is slowly consumed
+		for(var/obj/item/clothing/C in src.contents)
```

Comment:
[Implied src](https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md#legacy-code) is bad. `src.contents` -> `contents`